### DISCUSSION TITLE
ACE+TAO cross-compile, compiler parameter error

### DISCUSSION THREAD

**Discussion by ErhardLoh** [DOCGroup/ACE_TAO#1798] [Q&A]
I get a toolchain error when cross-compiling ACE+TAO with msys2 on Windows。
error message：arm-cortexa7hft-linux-gnueabi-g++.exe: error: unrecognized command line option '-mthreads'; did you mean '-pthread'?
Have you ever had a similar situation?

**Comment by mitza-oci** [comment]
Since this is not a configuration we typically test, you'll probably have to make changes to the `platform_*.GNU` files to get it to work.  You are essentially developing a new port.
